src/renderNothing.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default function () {
+  const major = React.version.split('.')[0];
```

Comment:
You really need to calculate it once. And you can check if it's < 15 only once too.